src/main/java/core/jdbc/JdbcTemplate.java

```diff
@@ -0,0 +1,68 @@
+package core.jdbc;
+
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JdbcTemplate {
+
+    public void update(String sql, Object... args) {
+        try (Connection con = ConnectionManager.getConnection();
+             PreparedStatement pstmt = con.prepareStatement(sql)) {
+            setArgs(pstmt, args);
+            pstmt.executeUpdate();
+        } catch (SQLException e) {
+            throw new JdbcTemplateException(e);
+        }
+    }
+
+    private void setArgs(PreparedStatement pstmt, Object[] args) throws SQLException {
+        for (int i = 0; i < args.length; i++) {
+            final int index = i + 1;
+            final Object value = args[i];
+
+            if (value instanceof String) {
+                pstmt.setString(index, (String) args[i]);
+            }
+            if (value instanceof Integer) {
+                pstmt.setInt(index, (Integer) args[i]);
+            }
```

Comment:
setObject 함수를 이용해보는 것은 어떨까요 :)
```suggestion
            pstmt.setObject(i + 1, args[i]);
```

Reply:
JDBC API 구현체의 setObject() 메서드  내부적으로 데이터 타입을 체크하는군요

덕분에 알았습니다. 감사합니당!!